opentelemetry-userevents-exporter/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "opentelemetry-userevents-exporter"
```

Comment:
we need to finalize a name. The existing exporters like zipkin don't have the word "exporter" in it.! Maybe we should do a bulk renaming to all existing crates! 

Not a blocker!

Reply:
Do we eventually want one or multiple crates to contain exporters to user events? If multiple, maybe rename this to "opentelemetry-user-events-logs"?

Reply:
Have renamed this to `opentelemetry-userevents-logs` for now. We can later decide if to merge.

Reply:
I think we definitely don't want to string together "userevents" in the name, should split those up as "user-events".

Reply:
Yes agree, have modified to `opentelemetry-user-events-logs`